### PR TITLE
Go/es

### PR DESCRIPTION
**Pull request by arranzropablo**
- [✓] I solemnly swear that this is all original content of which I am the original author
- [✓] Pull request title is prepended with `[language/lang-code]`
- [✓] Pull request touches only one file (or a set of logically related files with similar changes made)
- [✓] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [✓] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
- [✓] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@Menelion Could you please review this?

**Comment by Menelion** [comment]
@divayprakash Sure! :)

File: es-es/go-es.html.markdown [hunk 1]
```diff
@@ -45,7 +46,7 @@ import (
 	"io/ioutil" // Implementa algunas útiles funciones de E/S.
 	m "math"   // Biblioteca de matemáticas con alias local m.
 	"net/http" // Sí, ¡un servidor web!
-	"strconv"  // Conversiones de cadenas.
+	"strconv"  // Conversiones de strings.
```
Comment (Menelion) [inline es-es/go-es.html.markdown]: ¿Por qué?

File: es-es/go-es.html.markdown [hunk 2]
```diff
@@ -98,39 +99,39 @@ saltos de línea.` // mismo tipo cadena
 	// Sintaxis de conversión con una declaración corta.
 	n := byte('\n') // byte es un alias para uint8.
 
-	// Los Arreglos tienen un tamaño fijo a la hora de compilar.
-	var a4 [4]int           // Un arreglo de 4 ints, iniciados a 0.
-	a3 := [...]int{3, 1, 5} // Un arreglo iniciado con un tamaño fijo de tres
+	// Los arrays tienen un tamaño fijo a la hora de compilar.
```
Comment (Menelion) [inline es-es/go-es.html.markdown]: ¿Por qué? _Arreglo_ es la traducción correcta de _array_.

File: es-es/go-es.html.markdown [hunk 3]
```diff
@@ -56,35 +57,35 @@ func main() {
 	fmt.Println("¡Hola mundo!")
 
 	// Llama a otra función de este paquete.
-	másAlláDelHola()
+	masAllaDelHola()
```
Comment (Menelion) [inline es-es/go-es.html.markdown]: ¿Go no permite identificadores unicode?

**Comment by Menelion** [comment]
@arranzropablo Sorry, but I can't merge until I see your reasoning behind re-anglicizing of all of the programming terms. It's normal practice to translate them into target languages when speaking about them, so, say, _a string_ is _una cadena_ in Spanish, _строка_ in Russian, and so on. @holocronweaver @gitnacho Please give your thoughts as you did contribute to the Spanish translation of Go tutorial. Thanks!

File: es-es/go-es.html.markdown (hunk 3 above)
Comment (arranzropablo) [reply es-es/go-es.html.markdown]: Si, perdon, no sabia que permitia identificadores unicode

File: es-es/go-es.html.markdown (hunk 2 above)
Comment (arranzropablo) [reply es-es/go-es.html.markdown]: En mi opinion, no me parece una buena practica traducir los nombres de los tipos como array, string o slice, ya que por ejemplo no puedes declarar una variable como

`var foo cadena`

sino como

`var foo string`

File: es-es/go-es.html.markdown (hunk 1 above)
Comment (arranzropablo) [reply es-es/go-es.html.markdown]: https://github.com/adambard/learnxinyminutes-docs/pull/3484#discussion_r259733240

**Comment by holocronweaver** [comment]
I have limited experience with code not written in English, so I am not sure about best practice here.

While any experienced non-English speaking programmer will no doubt be familiar with many of the English technical terms (string, array, etc.), novice non-English speaking programmers may appreciate having at least the variable names in their native language in order to reduce the language barrier cognitive load.

Thus I tend to agree with @Menelion that we should retain localized variable names unless there is a good pedagogical reason not to.

**Comment by Menelion** [comment]
Yes sure, we can't declare a variable as type `cadena` nor `строка`, but that doesn't mean we should restrain ourselves from using local type names and terms, otherwise we'll get a tutorial (almost) in English. So, @divayprakash, I would close this. What's your opinion?

**Comment by divayprakash** [comment]
@Menelion I agree with your view on this.

**Pull request closed without merging**